wlauto/workloads/msword/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java

```diff
@@ -80,17 +78,15 @@ public void runUiAutomation() throws Exception {
         startLogger("welcome_screen_skip");
         clickUiObject(BY_TEXT, "Skip", true); // skip welcome screen
         stopLogger("welcome_screen_skip");
+        // Run the main tests
+        testExistingDocument(parameters.getString("document_name"));
```

Comment:
See comment in the other pull request.  The agenda should pass a param to enable the testing of an existing doc.  Then this workload will work OOB for typical WA users.